packages/itwinui-css/src/button/button-group.scss

```diff
@@ -15,8 +15,8 @@
     max-block-size: 100%;
   }
 
-  $childZindexSelector: '.iui-input-container, .iui-button, .iui-input:where(:not(.iui-input-container .iui-input))';
-  $childBorderSelector: '.iui-input, .iui-button:where(:not([data-iui-variant="borderless"]))';
+  $childZindexSelector: '.iui-input-flex-container, .iui-button, .iui-input:where(:not(input))';
```

Comment:
one of these selectors says `.iui-input:where(:not(input))` which does not make sense

maybe you mean `.iui-input:where(:not(.iui-input-grid .iui-input))`?

Reply:
Yup, changed